docs/changelog.md

```diff
@@ -17,12 +17,15 @@ permalink: /changelog/
 * **browser-icons**
   * Added `BrowserIcons.loadIntoView` to automatically load an icon into an `ImageView`.
 
+* **feature-downloads**
+  * Download filenames are now more accurate when `FetchDownloadManager` is used.
```

Comment:
nit: Could we add a bit more info for consumer to know why Download filenames are now more accurate? :)